hack/go-path.sh

```diff
@@ -16,4 +16,5 @@
 # enforce GOROOT
 # shellcheck disable=SC2155
 export GOROOT=$(go env GOROOT)
-export GOPATH=$(go env GOPATH)
\ No newline at end of file
+export GOPATH=$(go env GOPATH)
+export PATH=$GOROOT:$GOPATH/bin:$PATH
```

Comment:
I did it because even if I've got the GOPATH inside my PATH, I cannot find the `goimports` binary.
Are you experimenting this issue too or is it limited to my local setup?

Reply:
well, I have it in my .bashrc
I just don't think we should force the PATH